cluster.go

```diff
@@ -540,10 +546,12 @@ func (c *Cluster) run() {
 		c.pushPingMetrics(c.ctx)
 	}()
 
-	c.wg.Add(1)
+	c.wg.Add(len(c.informers))
 	go func() {
-		defer c.wg.Done()
-		c.pushInformerMetrics(c.ctx)
+		for _, informer := range c.informers {
+			defer c.wg.Done()
+			c.pushInformerMetrics(c.ctx, informer)
+		}
```

Comment:
Only one informer is getting launched here. You need to launch several goroutines, not one goroutine with a loop that hangs on the first iteration.